code/src/components/UserName.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable jsx-a11y/no-noninteractive-tabindex */
+/* eslint-disable react/no-unescaped-entities */
+import React from 'react';
+
+const UserName = ({ username, setUsername }) => {
+  const handleUserNameInputChange = (event) => {
+    setUsername(event.target.value);
+  };
+  return (
+    <div>
+      <p className="user-p" tabIndex="0"> Let's start with your first name.</p>
+      <input type="text" className="user-input" tabIndex="0" value={username} onChange={handleUserNameInputChange} placeholder="name" name="name" required />
+    </div>
```

Comment:
It seems like the required function isn't working properly, I wonder if you need to use `form` instead of a `div` to wrap your inputs and question-texts?